frontend/src/pages/Launch/index.tsx

```diff
@@ -68,23 +68,23 @@ export default function LaunchPage() {
           <Column gap="20">
             <Column gap="4">
               <Text.Body className={styles.inputLabel}>Name</Text.Body>
-              <Input placeholder="Dogecoin" {...register('name')} />
+              <Input placeholder="Starknet" {...register('name')} />
```

Comment:
the placeholder should not be Starknet and STRK

Reply:
made the changes